src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -25,6 +28,7 @@
 
     // Joystick for controlling the mechanisms of the robot
     private final Joystick controlStick;
+    
 
     public Joystick getDriverJoystick() {
```

Comment:
We need to completely rework the OI mappings on Friday, and I also did a little bit of  mapping in a different branch which well add in then